main/test/flix/Test.Exp.Regex.flix

```diff
@@ -0,0 +1,9 @@
+mod Test.Exp.Regex {
+
+    @test
+    def testRegex01(): Regex = Regex#""
+
+    @test
+    def testRegex02(): Regex = Regex#".*"
+
```

Comment:
I was expecting some more tests :)

Reply:
Am I free to use functions from Regex.flix at this point?

Reply:
Yes, I think so

Reply:
`Test.Exp.Regex` now has twelve tests.

`TestRegex.flix` in `library` has been updated to use regex literals except where we need to compile with flags or use the `unmatchable` expression.